config/app.php

```diff
@@ -104,7 +104,7 @@
     |
     */
 
-    'default_country' => null,
+    'default_country' => 'DE',
```

Comment:
I think you forgot to set this back to null ?

Reply:
or to 'en' ?

Reply:
I reverted my changes and added some information about this variable. Also I added some workaround in tests so they won't fail, if default_country is set to null.